roscala/src/main/scala/coop/rchain/rosette/Op.scala

```diff
@@ -8,12 +8,12 @@ case class OpNargs(n: Int) extends Op
 case class OpAlloc(n: Int) extends Op
 case class OpPushAlloc(n: Int) extends Op
 case class OpExtend(v: Int) extends Op
-case class OpOutstanding(p: Int, n: Int) extends Op
+case class OpOutstanding(pc: Int, n: Int) extends Op
 case class OpFork(p: Int) extends Op
 case class OpXmitTag(u: Boolean, n: Boolean, m: Int, v: Int) extends Op
```

Comment:
In every case with both u and n, u means unwind and n means next.  If you're making the change here, please change it everywhere.

Reply:
Sorry, just realized this now after seeing the changes: `n` is not `next` in every case. For `OpNargs` it would be `nargs`. For `OpAlloc` and `OpPushAlloc` it can stay `n` and so on.

Reply:
Can it be `next` for `OpAlloc`?

Reply:
No, `next` would not make sense for `OpAlloc`. I would also leave that to `n`.

Reply:
> `n` is not `next` in every case. For `OpNargs` it would be `nargs`.

That's why I said "with both u and n".